examples/hpc-enterprise-slurm-v6.yaml

```diff
@@ -0,0 +1,330 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+blueprint_name: hpc-enterprise-slurm
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: hpc01
```

Comment:
I think we should use deployment name more consistent with the blueprint. wdyt ?

Reply:
I don't like this name, but I would prefer to keep it the same as V5 blueprint does.

Reply:
I would prefer from v6+ we use consistent name as v5 would be removed sooner or later. But yeah not a strong opinion.